Connected/Assets/Scripts/Components/Battery.cs

```diff
@@ -4,14 +4,10 @@
 
 public class Battery : PowerSource {
     [SerializeField]
-    float Voltage = 12f;
-
-    [SerializeField]
-    float Ampere = 1f;
+    float ampere = 1f;
```

Comment:
Sorry if this was unclear, but the ampere will not be set by the battery, but rather as part of the circuit tracing, so this should probably not be a serialized field.

Reply:
You were clear, I just forgot about it and was simply not careful enough. Thanks for spotting yet another mistake. I know how the current is calculated, but simply didn't reflect about it when changing the name. :/